search-sentiments-map/public/scripts/current-data-script.js

```diff
@@ -35,19 +35,13 @@ function getCurrentCustomSearchData() {
  * @param {string} trend New trend to get data for.
  */
 function setNewTrend(trend) {
-  // TODO(carmenbenitez): Add if/else block to show data for other search
-  // results when custom search data for all current top trends is set up. 
-  currentTrend = 'The Old Guard';
-  
-  const topicHeader = document.getElementById('topic-header');
-  topicHeader.innerText = 
-      'Worldwide sentiments of search results for "' + currentTrend + '"';
+  currentTrend = trend;
 
-  // Reload map with new sentiment data and relevant coloring.
-  fetch('/search/' + currentTrend)
-      .then(resultsJsonArray => resultsJsonArray.json()).then(topicResults => {
-        currentCustomSearchData = topicResults;
-        loadCountryData();
-      });
+  fetch('/search/' + trend)
```

Comment:
Should use currentTrend and not trend.

Reply:
Wouldn't those two be the same tho?

Reply:
Yes but its better to use the currentTrend I would think since that is the general variable for it.

Reply:
Since it's in the function where trend is a parameter, I'd say it's better to use trend